src/bp/core/config/bot.config.ts

```diff
@@ -86,4 +86,9 @@ export interface ConverseConfig {
    * @default 250
    */
   bufferDelayMs: number
+  /**
+   * Whether or not you want to expose public converse API. See docs here https://botpress.com/docs/channels/converse#public-api
+   * @default ture
+   */
+  enableUnsecuredEndPoint: boolean
```

Comment:
Shouldn't we add an "enabled" field for both types? (secured and unsecured). Maybe we could name the field  so we don't need to update current configs ? 
```js
disable: { 
  secured: false, 
  unsecured: true 
} 
```

Reply:
I don't see why one would disable the authenticated converse endpoint ? Is there and actual real reason you're suggesting this ?

Reply:
Right. let's keep it that way